src/Error/Middleware/ErrorHandlerMiddleware.php

```diff
@@ -121,6 +133,10 @@ public function handleException($exception, $request, $response)
      */
     protected function getRenderer($exception)
     {
+        if (!$this->renderer) {
+            $this->renderer = $this->config('exceptionRender') ?: ExceptionRenderer::class;
+        }
```

Comment:
Good point, but that would introduce strings again, which is what I think @dereuromark is trying to remove as well.